src/cargo/core/compiler/timings.rs

```diff
@@ -732,7 +736,7 @@ h1 {
 </head>
 <body>
 
-<h1>Cargo Build Timings</h1>
+<h1>Cargo Build Timings <a href="https://doc.rust-lang.org/nightly/cargo/reference/timings.html">Documentation</a></h1>
```

Comment:
```suggestion
<h1>Cargo Build Timings</h1>
<a href="https://doc.rust-lang.org/nightly/cargo/reference/timings.html">Documentation</a>
```

Reply:
Just for context, we were chatting about this change and felt like putting it just below the header might look a little better while still making it easily discoverable.

Reply:
![image](https://user-images.githubusercontent.com/29879298/163180843-152b032d-8276-4f93-a5b8-f492bcc6c8d5.png)